### DISCUSSION TITLE
Mobile version of GitHub.com accidentally truncates branches list

### DISCUSSION THREAD

**Discussion by LinqLover** [community/community#4166] [Mobile]
Originally reported in https://github.com/isaacs/github/issues/1975:

> ![image](https://user-images.githubusercontent.com/38782922/118288553-d5ebd100-b4d4-11eb-9180-50128824998c.png)
> 
> On the desktop, you can see that the actual branch list is actually larger. Even if I rotate my phone the list grows.

**Comment by alcere** [comment]
Hi! Thanks for reporting this big. I created an internal issue to get it fixed as soon as possible!
